sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/http/ReactorNettyHttpClientTest.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.http;
+
+
+import com.azure.cosmos.implementation.Configs;
+import com.azure.cosmos.implementation.LifeCycleUtils;
+import com.azure.cosmos.implementation.directconnectivity.ReflectionUtils;
+import io.netty.channel.ChannelOption;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests that partition manager correctly resolves addresses for requests and does appropriate number of cache refreshes.
+ */
+public class ReactorNettyHttpClientTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(ReactorNettyHttpClientTest.class);
+    private Configs configs;
+    private HttpClient reactorNettyHttpClient;
+
+    @BeforeClass(groups = "unit")
+    public void before_ReactorNettyHttpClientTest() {
+        this.configs = new Configs();
+        this.reactorNettyHttpClient = HttpClient.createFixed(new HttpClientConfig(this.configs));
+    }
+
+    @AfterClass(groups = "unit")
+    public void after_ReactorNettyHttpClientTest() {
+        LifeCycleUtils.closeQuietly(reactorNettyHttpClient);
+    }
+
+    @Test(groups = "unit")
+    public void testHttpClientWithMaxHeaderSize() {
```

Comment:
as per our code style, we do not use "test" prefix for any test method.

please remove the 'test' prefix for this method and all other test methods in this file.

Reply:
@moderakh  - thanks for pointing that out. That was actually added by me. Removed the prefix now. :)